lesson07/assignment/vue-cli/src/views/About.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+
+<div role="about">
+<div v-for="(stuff, idx) in accordionObject" :key='idx' class="panel panel-default">
+
+  <b-card no-body class="mb-1">
+    <b-card-header header-tag="header" class="p-1" role="tab">
+      <b-button block href="#" v-b-toggle="'accordion-'+idx" variant="info">{{stuff.title}}</b-button>
+    </b-card-header>
+    <b-collapse :id="'accordion-'+idx" visible accordion="my-accordion" role="tabpanel">
```

Comment:
Excellent job assigning unique IDs to each accordion tab and selecting appropriately in the button.